stubs/docutils/docutils/nodes.pyi

```diff
@@ -1,15 +1,75 @@
+import xml.dom.minidom
 from _typeshed import Self
-from collections.abc import Iterable
-from typing import Any, overload
+from collections.abc import Iterable, Sequence
+from typing import Any, Callable, Generator, Protocol, TypeVar, overload
+from typing_extensions import Literal
 
 from docutils.transforms import Transformer
 
+class NodeVisitor:
```

Comment:
This class should be moved below the "type stuff", ideally about the same position where it resides in the implementation.